### ISSUE TITLE
Final version of code

### ISSUE THREAD

**Issue by ATakaSKY** [StephenGrider/MLKits#3]
Well put up course!

Just wanted to ask you to post the final solutions as well on the repo or on the udemy content list if possible. It sometime gets hard to track each and every thing as the course involves a lot of intricate stuffs. 

I am currently having issue in the linear regression gradient descent part where we implement our own learning rate algorithm. Although i have followed as per your code but i am not getting expected results.

Would be a great help if you could post solutions if not here atleast on the udemy course.
Thanks.

**Comment by kieran1985** [comment]
+1 for this, any chance?

**Comment by ethan-fang** [comment]
+1 too. I'm learning lec153 and did exactly the same change as the video but my accuracy is still 0.08 instead of 0.87. It's quite frustrating because it's quite hard to debug. Sample for each section would be very useful.

**Comment by ethanolle** [comment]
+1 too.

**Comment by oded-regev** [comment]
+1 @StephenGrider
